lib/ansible/plugins/lookup/consul_kv.py

```diff
@@ -99,8 +103,7 @@
 from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ansible.errors import AnsibleError, AnsibleAssertionError
 from ansible.plugins.lookup import LookupBase
-
-import json
+from ansible.module_utils._text import to_native
```

Comment:
I suspect you actually want 'to_text' as bytestrings are not going to be very usable in jinja2 context

Reply:
Right you are. Fixed it